src/Models/Node.ts

```diff
@@ -25,5 +26,7 @@ export class Node extends Model {
                 sequelize: sequelize,
                 tableName: "Nodes",
             } as InitOptions);
+        this.belongsTo(User, {foreignKey: "userId"});
+        User.hasMany(Node, {foreignKey: "userId"});
```

Comment:
This is weird to be here, should be better if in User model as it adds new properties to User model, not Node.